tensorbay/client/segment.py

```diff
@@ -176,19 +176,17 @@ def _put_binary_file_to_azure(
 
     def _synchronize_upload_info(  # pylint: disable=too-many-arguments
         self,
-        remote_path: str,
-        checksum: str,
-        frame_info: Optional[Dict[str, Any]] = None,
+        callback_info: Sequence[Dict[str, Any]],
         skip_uploaded_files: bool = False,
     ) -> None:
         put_data: Dict[str, Any] = {
             "segmentName": self.name,
-            "objects": [{"checksum": checksum, "remotePath": remote_path}],
+            "objects": [],
```

Comment:
`"objects": callback_info`?

Reply:
I will contact with the backend engineer later.